easybuild/easyconfigs/s/samblaster/samblaster-v.0.1.24-foss-2018b.eb

```diff
@@ -0,0 +1,28 @@
+#  easybuild easyconfig
+#
+#  John Dey jfdey@fredhutch.org
+#  Fred Hutchinson Cancer Research Center
+easyblock = 'MakeCp'
+
+name = 'samblaster'
+version = 'v.0.1.24'
```

Comment:
@fizwit Should we correct this to a more standard `0.1.24`, and then change the `source_urls` and `sources` accordingly?

```python
version = '0.1.24'

...

source_urls = ['https://github.com/GregoryFaust/samblaster/releases/download/v.%(version)s']
sources = ['%(name)s-v.%(version)s.tar.gz']
```

Reply:
fix version and use download_filename